docs/docs/api/09-Remote-Device-Controller.md

```diff
@@ -0,0 +1,114 @@
+
+2. Start the installer and follow the installation process.
+    > Note: Installation requires administrator rights.
+
+3. Create a new AskUI project as [described in our documentation](../general/02-Getting%20Started/getting-started.md).
+
+4. Open file `test/helper/jest.setup.ts` in the AskUI project.
+
+5. Remove `await uiController.start();` in `beforeAll` function.
+
+6. Remove `await uiController.stop();` in `afterAll` function.
+
+7. Start the installed UI Controller manually and select the screen you want to use. 
+    > Note: Either through the start menu, desktop shortcut or the executable in the install directory.
+
+8. You should be good to go now to run your workflows as described in documentation.
+    > Note: Ensure that the UI Controller is running before you start a workflow.
+
+### Manual Update of Remote Device Controller
+
+Make use of our preview build to use the latest features by following the mentioned steps below.
+
+1. Go to function `beforeAll` there you must modify in file `test/helper/jest.setup.ts`.
+
+2. Ensure that you have the required import `UiController`.
+
+        import { UiController } from 'askui';
+
+3. Also add the global variable if it is missing:
+
+        let uiController: UiController;
+
+4. Modify the creation of `uiController` in file `test/helper/jest.setup.ts`:
+
+        uiController = new UiController({
+          /**
+            * Select the display you want to run your tests on, display 0 is your main display;
+            * ignore if you have only one display
+            */
+          display: <YOUR_DISPLAY_ID>,
+          binaryVersion: "tdk-23.10.1-preview-release-1"
+        });
+
+   > Note: The ui controller creation starts typically with `uiController = await UiController({ ...` but the instance name might be different.
+
+5. Replace `<YOUR_DISPLAY_ID>` with the display (starting with 0 for the first display) that you want to use:
+
+        display: <YOUR_DISPLAY_ID>,
+
+6. Ensure that the controller gets started before the UI controller client tries (`aui`) to connect. Therefore the last two lines of the function should do:
+
+       await uiController.start();
+       await aui.connect();
+
+    > Note: The ui controller client creation starts typically with `aui = await UiControlClient.build({ ...` but the instance name might be different.
+
+7. Ensure that in function `afterAll` the controller gets stopped after the ui controller client closes the connection:
+
+        aui.close();
+        await uiController.stop(true);
+
+8. You should be good to go now to run your workflows as described in documentation.
+
+> Note: New features are only supported on Windows 10/11.
+
+## Features
+
+### Type Like a Human
+Together wit the asynchronous execution of actions we also introduce natural typing. The 
+first iteration of this feature uses the typical typing speed of a human.
+
+### Experimental Unicode Support
+Typing Unicode text is supported but is not yet production ready.
```

Comment:
Important to the user? The whole thing is not yet "production ready".

Reply:
I will shorten it to the heading only. Makes more sense.